### DISCUSSION TITLE
Is it okay to add exclusion temporarily to net10?

### DISCUSSION THREAD

**Discussion by nohwnd** [dotnet/source-build#4786] [Q&A] [answered]
https://github.com/dotnet/sdk/runs/33903175722 

We are piloting here a new experience for `dotnet test` and the dependency we are using is coming from microsoft/testfx that is not enrolled in source build. The feature is hidden below a feature flag and is opted-out by default. Is it okay to add an exclusion to prebuilts.xml to be able to merge, while we figure out how to do this properly ? (e.g. move to code to dotnet/sdk. or enroll testfx into source build)

Is there an easy way for us to enroll just one project in the repo to source build in case we would decide to add the testing.platform component to source build?

**Comment by MichaelSimons** [comment]
Do you have a timeframe in mind to resolve this prebuilt?  If it would be resolved by Preview 1 I would entertain taking it.  Given the relatively short timeline given the upcoming holidays to Preview 1, I would want a plan in place to addressing the prebuilt.  If you would like to discuss the options available feel free to schedule a meeting.

**Reply by nohwnd** [reply]
I don't think it would be that fast given the holidays are coming, I will try to setup a call, but for now it is probably the best for use to copy the code into sdk. It is just a few classes.

**Reply by nohwnd** [reply]
Okay it was easier than I thought it will be to isolate just one project (thanks to help from Viktor). But now I am seeing a lot of pre-builts for things that are build related, and that we don't ship. The nuget itself only ships a single dll, and has no dependencies.

https://dev.azure.com/dnceng-public/public/_build/results?buildId=890110&view=logs&j=2f0d093c-1064-5c86-fc5b-b7b1eca8e66a&t=52d0a7a6-39c9-5fa2-86e8-78f84e98a3a2

**Reply by MichaelSimons** [reply]
@nohwnd - can you link to the PR/branch where this work is happening?  The build link appears to have expired already.

**Reply by nohwnd** [reply]
https://github.com/microsoft/testfx/runs/34137204187 this is my PR. I am trying to sync the arcade to what we have in vstest, to see if that passes the build. Plus there is external dependency on Polyfill. I added that to exclusions, but I don't know if that is the right thing to do. And disabled analyzers for source build like we do in vstest.

**Reply by nohwnd** [reply]
https://github.com/microsoft/testfx/pull/4295 wrong link

**Comment by nohwnd** [comment, marked as answer]
We enrolled.
